docs/integrations/source_version_control.md

```diff
@@ -7,8 +7,12 @@ Use [pre-commit](https://pre-commit.com/). Once you
 ```yaml
 repos:
   - repo: https://github.com/psf/black
-    rev: stable # Replace by any tag/version: https://github.com/psf/black/tags
+    rev: 21.7b0 # Replace by any tag/version: https://github.com/psf/black/tags
```

Comment:
We'll have to put something in the release process to update this (or better yet, make it automatic).

Reply:
sure, I'll add a pre-commit check for this

Reply:
I've added a check to ensure that the latest version from `CHANGES.md` matches the rev in the example from `source_Version_control.md`

demo:
```console
(venv) marco@marco-Predator-PH315-52:~/black-dev$ git diff
diff --git a/docs/integrations/source_version_control.md b/docs/integrations/source_version_control.md
index b63a7c4..64cc8b1 100644
--- a/docs/integrations/source_version_control.md
+++ b/docs/integrations/source_version_control.md
@@ -7,7 +7,7 @@ Use [pre-commit](https://pre-commit.com/). Once you
 ```yaml
 repos:
   - repo: https://github.com/psf/black
-    rev: 21.7b0 # Replace by any tag/version: https://github.com/psf/black/tags
+    rev: 21.6b0 # Replace by any tag/version: https://github.com/psf/black/tags
     hooks:
       - id: black
         language_version: python3 # Should be a command that runs python3.6+
(venv) marco@marco-Predator-PH315-52:~/black-dev$ git commit -a -m 'foo'
black................................................(no files to check)Skipped
Check pre-commit rev in example is correct...............................Failed
- hook id: check-pre-commit-rev-in-example
- exit code: 1

Traceback (most recent call last):
  File "/home/marco/miniconda3/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/home/marco/miniconda3/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/marco/black-dev/scripts/check_pre_commit_rev_in_example.py", line 32, in <module>
    main(changes, source_version_control)
  File "/home/marco/black-dev/scripts/check_pre_commit_rev_in_example.py", line 19, in main
    raise ValueError(
ValueError: Please set the rev in ``source_version_control.md`` to be the latest one

flake8...............................................(no files to check)Skipped
mypy.................................................(no files to check)Skipped
prettier.................................................................Passed
Fix End of Files.........................................................Passed
Trim Trailing Whitespace.................................................Passed
```